### DISCUSSION TITLE
Population size

### DISCUSSION THREAD

**Discussion by heatherturtle** [lvclark/polyRAD#34] [General]
Hi Lindsay, 

Can polyRAD be used for a smaller population of 60, or is it best suited for larger pops?

**Comment by lvclark** [comment]
It is best suited for larger pops, since estimates of allele frequency won't be very accurate in a population of that size (unless it is totally unstructured).  An alternative is you can use the "naïve method" described here: https://lvclark.r-universe.dev/articles/polyRADtutorials/population_genetics.html.  That would ignore allele frequencies entirely.
